relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py

```diff
@@ -183,6 +183,13 @@ def download(completeUrl, dst=""):
            fail("Received a HTTP %s error when trying to get %s"%(r.status_code, completeUrl))
     return fileDst
 
+"""
+    Make a HEAD request on the given url, return true if result is 200, false instead
+"""
+def check_download(completeUrl):
+    r = requests.head(completeUrl, auth=(USERNAME, PASSWORD))
+    return (r.status_code < 400)
```

Comment:
why not < 300 ?

Reply:
We are doing redirection, so 301 is ok.
Could be <= 301